pages/concepts/bank-funding-source.mdx

```diff
@@ -0,0 +1,149 @@
+---
+layout: concepts
+title: "Bank Funding Source"
+description: Learn about the different methods of adding and verifying a bank funding source to a Customer account.
+category: Funding Sources
+concept:
+  icon: "bank-funding-source.svg"
+meta:
+  title: Bank Funding Source | Dwolla API Documentation
+  description: A bank funding source represents a bank account attached to a Customer resource or the Master Account which can be used to create ACH transfers.
+---
+
+import addFundingSourceForm from "../../assets/images/content-images/add-a-funding-source-ui-form.png";
+import guideIcon from "../../assets/images/content-images/content-icons/guides-icon.svg";
+import iavForm from "../../assets/images/content-images/iav-form.png";
+import plaidLink from "../../assets/images/content-images/plaid-link.png";
+
+# Bank Funding Source
+
+There are many different ways for a Customer to add a bank account on Dwolla’s platform. Choosing the approach that is ideal for your application depends on a number of factors, such as speed and/or user experience. As you think about the different method(s) to [attach a bank account](https://www.dwolla.com/platform/bank-verification-methods/), you will need to consider which one best suits the use case of your business’ application.
```

Comment:
I wonder if we should remove the link here since this is specifically referencing the methods to "attach" a bank. The link is referencing verification